indico/modules/events/contributions/forms.py

```diff
@@ -32,3 +33,4 @@ class ContributionForm(IndicoForm):
     duration = TimeDeltaField(_("Duration"), [DataRequired()], default=timedelta(minutes=20),
                               units=('minutes', 'hours'),
                               description=_("The duration of the contribution"))
+    location_data = IndicoLocationField(_("Location"), widget=LocationWidget(), description=_("Location"))
```

Comment:
You can set `widget = LocationWidget()` in the `IndicoLocationField` class, no need to specify it here if it's the default one for the field